aspnetcore/signalr/version-differences.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Differences between SignalR and SignalR Core
+author: rachelappel
+description: Differences between SignalR and SignalR Core
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/30/2018
+uid: signalr/version-differences
+---
+# Differences between SignalR and ASP.NET Core SignalR
+
+SignalR for ASP.NET Core is not compatible with previous versions of SignalR. This means that you cannot use the old server with the new clients or the old clients with the new server. The following article details the features which have been removed or changed in the ASP.NET Core version of SignalR.
```

Comment:
Non pre-release version mismatches between server and client are okay **within** ASP.NET Core SignalR. But you can't mix ASP.NET Core SignalR and ASP.NET SignalR on the client and server.